src/page-support/exclusive-content/components/generated-info-text.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import SyntaxHighlighter from 'react-syntax-highlighter'
+import { docco } from 'react-syntax-highlighter/dist/esm/styles/hljs'
+import { CopyContent } from './copy-content'
+
+export function GeneratedInfoText() {
+  const divTag = `<div id="ec">
```

Comment:
we might want to use an id that's generated for each piece of content, in case we want several on one page

Reply:
Good point. I'll use classes and generated ids.